actions/actions.py

```diff
@@ -549,3 +551,18 @@ async def run(
         events.append(ActionExecuted("action_listen"))
 
         return events
+
+
+class ActionRestart(Action):
+    def name(self) -> Text:
+        return "action_restart"
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[EventType]:
+
+        return [Restarted(), FollowupAction("action_session_start")]
+       
```

Comment:
Linting error from the extra spaces on the last line.

Reply:
urg I should add a hook to the repo so it blacks things on commit, I do this too often